setup.py

```diff
@@ -202,9 +202,13 @@
 this_directory = Path(__file__).parent
 long_description = (this_directory / "README.md").read_text()
 
+# current version
+with open(os.path.join("pedalboard", "version.txt")) as f:
+    version = f.read().strip()
+
```

Comment:
Sticking to PEP documentation, let's do something like this:

```python
# read version
version = {}
with open("pedalboard/version.py") as fp:
    exec(fp.read(), version)
```

Reply:
I see, working on it